project/Common.scala

```diff
@@ -152,9 +170,9 @@ object Common {
     unmanagedResourceDirectories in Test +=
       (baseDirectory in Build.root).value / "resources/test",
 
-    publishTo := Some("Spark Kernel Nexus Repo" at repoUrl),
-
-    credentials += Credentials("Sonatype Nexus Repository Manager", repoHost, repoUsername, repoPassword),
```

Comment:
I would call this a different name, I actually got confused, thinking you were directly publishing to Sonatype, while you are just publishing to the Apache staging repository